pkg/agent/handler/accessrequest_test.go

```diff
@@ -228,7 +228,7 @@ func TestAccessRequestHandler(t *testing.T) {
 					},
 				},
 			}
-			handler := NewAccessRequestHandler(arp, cm, c)
+			handler := NewAccessRequestHandler(arp, cm, c, config.DiscoveryAgent)
```

Comment:
Can you please add a test for the trace agent resource handling

Reply:
Done